frontend/amundsen_application/static/js/components/Alert/AlertList.tsx

```diff
@@ -10,19 +10,48 @@ export interface AlertListProps {
   notices: NoticeType[];
 }
 
+export interface AggregatedAlertListProps {
+  notices: {
+    [key: string]: NoticeType;
+  };
+}
+
+const aggregateNotices = (notices) =>
+  notices.reduce((accum, notice: any) => {
```

Comment:
Any way to avoid this 'any'?

Reply:
Yeah should be able to, might've just been leftover from my first implementation :)

Reply:
Mapped it to `NoticeType`